approval_polls/views.py

```diff
@@ -79,7 +79,7 @@ def add(request):
     if not len(choices):
       return render(request, 'approval_polls/error.html', {'error_message': 'At least one choice is required'})
 
-    p = Poll(question=question, pub_date=timezone.now())
+    p = Poll(question=question, pub_date=timezone.now(), username=request.POST['username'].strip())
```

Comment:
@ashimaathri 
I did not realize we can associate other object as a field.  Sorry for not understanding very well.
I will make changes and send PR soon.
Thank you!